### PR TITLE
Give Campaigns Type and Tests

### PR DESCRIPTION
**Pull request by delianides**

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://1c3037e4-7280-4d2c-8108-c513d29f7537.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzmann/branch/give-campaigns-refactor/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

File: imports/pages/give/campaign/index.js [hunk 1]
```diff
@@ -89,12 +90,13 @@ class Template extends Component {
   }
 }
 
+export default withFinancialAccounts(connect()(Template));
```
Comment (johnthepink) [inline imports/pages/give/campaign/index.js]:
```js
const TemplateWithData = withFinancialAccounts(connect()(Template));
```

File: imports/pages/give/campaign/index.js [hunk 2]
```diff
@@ -89,12 +90,13 @@ class Template extends Component {
...
 
 const Routes = [
   { path: "campaign/:name", component: Template },
 ];
 
-export default {
+export {
```
Comment (johnthepink) [inline imports/pages/give/campaign/index.js]: back to default

File: imports/pages/give/index.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
-
+// XXX with flow this might need to be refactored
 import Campaign from "./campaign";
 import History from "./history";
```
Comment (johnthepink) [inline imports/pages/give/index.js]:
```js
import { Routes as CampaignRoutes } from "./campaign";
```

File: imports/pages/give/index.js [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
 import Now from "./now";
 import Review from "./review";
 import Schedules from "./schedules";
 
+/* eslint-disable */
 export default {
```
Comment (johnthepink) [inline imports/pages/give/index.js]: if this does need to be disabled

```
// eslint-disable-next-line name-of-lint-error
```

File: .types/lodash-find.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+declare module "lodash.find" {
+  declare var exports: any;
+};
```
Comment (johnthepink) [inline .types/lodash-find.js]: let's try to use the default export declaration like yay:

```js
declare module "lodash.find" {
  declare export default find(list: any[]);
}
```

but w/e the correct version should be

File: imports/pages/give/campaign/index.js [hunk 6]
```diff
@@ -1,10 +1,12 @@
-/* eslint-disable import/no-named-as-default */
-import { Component, PropTypes } from "react";
+// @flow
+import { Component } from "react";
 import { connect } from "react-redux";
 import { graphql } from "react-apollo";
 import gql from "graphql-tag";
 
+/* eslint-disable */
```
Comment (johnthepink) [inline imports/pages/give/campaign/index.js]: if needs to be disabled:
```
// eslint-disable-next-line name-of-lint-error
```

**Comment by delianides** [comment]
@johnthepink fixed. Verified this works by running `--native`

**Pull request merged**